web/src/controllers/InstallerAPI.ts

```diff
@@ -101,7 +101,7 @@ export class Installers {
     }
     i.id = i.hash();
 
-    const err = await i.validate();
+    const err = await i.resolve().validate();
```

Comment:
Convert `latest` to actual version so the Rook/Kubernetes version validator succeeds.